rust/main/hyperlane-base/src/settings/trace/mod.rs

```diff
@@ -76,11 +76,12 @@ impl TracingConfig {
                 .with_target("tendermint", Level::Info)
                 .with_target("tokio", Level::Debug)
                 .with_target("tokio_util", Level::Debug)
-                .with_target("aws_sdk_s3", Level::Info)
-                .with_target("aws_smithy_runtime_api", Level::Info)
                 // Enable Trace level for Tokio if you want to use tokio-console
                 // .with_target("tokio", Level::Trace)
                 // .with_target("tokio_util", Level::Trace)
+                .with_target("aws_sdk_s3", Level::Info)
+                .with_target("aws_smithy_runtime", Level::Info)
+                .with_target("aws_smithy_runtime_api", Level::Info)
```

Comment:
i think `aws_smithy_runtime` will match `aws_smithy_runtime_api` as well, since it matches the prefix iiuc. But not sure

Reply:
Presumably, yes, they talk about prefix in the docs, but in example it is always "my_crate" and never "my_cra".

Reply:
It works with random substrings.

Reply:
I pushed a commit.